frontend/apps/web/app/(mgmt)/[account]/jobs/util.ts

```diff
@@ -1765,6 +1766,7 @@ export async function validateJobMapping(
       });
     }),
     connectionId: connectionId,
+    jobSource: jobSource,
```

Comment:
how is the job source different from the `connectionId`? aren't they the same thing? Could `connectionid` go away in favor of just the job source?

Reply:
I still have this question

Reply:
I wanted to keep the validate job mapping endpoint flexible. It doesn't require the jobsource. If you just want to validate the mappings then all you need is mappings and connection_id. Like you could call it with the mappings and destination id and it will tell you all the missing tables

Reply:
sounds good